src/current/_includes/releases/v22.2/v22.2.12.md

```diff
@@ -0,0 +1,22 @@
+## v22.2.12
+
+Release Date: July 24, 2023
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v22-2-12-sql-language-changes">SQL language changes</h3>
+
+- The `debug doctor` command now allows you to verify that primary indexes store all columns. [#106936](https://github.com/cockroachdb/cockroach/pull/106936)
+- Attempting to drop index that is the only source of a column is now forbidden and results in an error. [#106936](https://github.com/cockroachdb/cockroach/pull/106936)
+
+<h3 id="v22-2-12-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug in the new [declarative schema changer](../v23.1/online-schema-changes.html#declarative-schema-changer) framework that could cause the incorrect retrieval of the unique secondary index rather than the primary index. This could cause a schema change to fail. For details and to determine whether a cluster is impacted, refer to [Technical Advisory 99561](https://cockroachlabs.com/docs/advisories/a99561). [#106428](https://github.com/cockroachdb/cockroach/pull/106428)
```

Comment:
```suggestion
- Fixed a bug in the new [declarative schema changer](../v23.1/online-schema-changes.html#declarative-schema-changer) framework that could cause the incorrect retrieval of the unique secondary index rather than the primary index. On such a cluster, an ALTER TABLE..ADD COLUMN statement will result in data for the new column being added to the secondary index resulting in query inconsistency. For details and to determine whether a cluster is impacted, refer to [Technical Advisory 99561](https://cockroachlabs.com/docs/advisories/a99561). [#106428](https://github.com/cockroachdb/cockroach/pull/106428)
```